src/SpecialFunctions.jl

```diff
@@ -91,7 +90,9 @@ include("gamma.jl")
 include("gamma_inc.jl")
 include("betanc.jl")
 include("beta_inc.jl")
-include("chainrules.jl")
+if !isdefined(Base, :get_extension)
```

Comment:
Do I understand correctly this PR is basically removing functionality for Julia v1.8-?

Reply:
That'd be pretty unfortunate :p.

Reply:
for Julia 1.8- my understanding is that it is the same as the old behavior (i.e. it loads ChainRulesCore unconditionally?)